lightningd/opening_control.c

```diff
@@ -223,7 +223,9 @@ wallet_commit_channel(struct lightningd *ld,
 			      true,
 			      &uc->local_basepoints,
 			      &uc->local_funding_pubkey,
-			      NULL);
+			      NULL,
+			      /* Channel specific feerates are disabled with 0 */
+			      0, 0);
```

Comment:
But zero is a valid value, which is a problem!  We should simply initialize the values to the globals if not set.

Reply:
Thanks, will find a way to allow zero then, this was an open question to me that was not addressed yet.